### PR TITLE
Add YouClap 👏🚀

### PR DESCRIPTION
**Pull request by portellaa**

### CODE REVIEW

**Comment by Couto** [comment]
buuuuuh!

**Comment by portellaa** [comment]
@Couto get out, stop being social 😂

**Comment by marmelo** [comment]
I am scared of approving this pull request.  
But now I am curious about what will happen!

**Comment by marmelo** [comment]
Thanks @portellaa!

**Comment by Couto** [comment]
Hey @portellaa it seems you just got your
























[](https://www.youtube.com/watch?v=6YMPAH67f4o) <- and you can actually click here!

File: README.md [hunk 1]
```diff
@@ -50,5 +50,6 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [WeDo Technologies](https://www.wedotechnologies.com/) [:rocket:](https://www.wedotechnologies.com/en/careers/) | Revenue Assurance and Fraud Management solutions. | `Braga` `Lisboa` |
 | [WIT Software](https://www.wit-software.com) [:rocket:](https://www.wit-software.com/careers/) | Mobile telecommunications services. | `Aveiro` `Coimbra`<br>`Leiria` `Lisboa`<br>`Porto` |
 | [YLD](https://www.yld.io/) [:rocket:](https://www.yld.io/join-us/) | Software engineering, design, training and open-source. | `Lisboa` `Porto`<br>`Remote` |
+| [YouClap](https://youclap.tech) | Create, participate and share awesome challenges. | `Aveiro` `Remote` |
```
Comment (filipalacerda) [inline README.md]: Como assim, remote? 😛😂

Comment (portellaa) [reply README.md]: you can work from the balcony in our office 😂 or from wherever you feel more comfortable 😜

**Comment by portellaa** [comment]
thanks @marmelo ❤️ 

Also @Couto for all the marketing you did 😘

File: README.md (hunk 1 above)
Comment (filipalacerda) [reply README.md]: 😂😂

Comment (marmelo) [reply README.md]: :thinking:

**Comment by Couto** [comment]
> Also @Couto for all the marketing you did 😘

Actually you own me a :beer: ! It's pretty hard to come up with dead jokes like the one above! :joy:

File: README.md (hunk 1 above)
Comment (portellaa) [reply README.md]: 🙃 i'm kidding 😂

**Comment by portellaa** [comment]
> > Also @Couto for all the marketing you did 😘
> 
> Actually you own me a 🍺 ! It's pretty hard to come up with dead jokes like the one above! 😂

Just let me know when 😜

**Pull request merged**